src/components/MovieCard/MovieCard.js

```diff
@@ -1,42 +1,75 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import ThreeDotsMenu from '../ThreeDotsMenu/ThreeDotsMenu';
+import { getYearFromDate } from '../../utils/utils';
+import { useHover } from '../../hooks/hooks';
```

Comment:
Why do we need this hook, we could just use onMouseEnter, onMouseLeave

Reply:
Just for reusability and training. I googled that this is one of the used ones and decided to redo this functionality into a hook.